cmd/kool_task.go

```diff
@@ -20,13 +20,13 @@ type KoolTask interface {
 // DefaultKoolTask holds data for running kool service as a long task
 type DefaultKoolTask struct {
 	KoolService
-	message string
-	taskOut shell.OutputWriter
+	message   string
+	taskShell shell.Shell
```

Comment:
any reason to not call the this simply `shell` instead of `taskShell`?

Reply:
kool task has two shells...one capture the original service output....the other handles the terminal...